src/sentry/monitors/endpoints/organization_monitors.py

```diff
@@ -219,4 +221,47 @@ def post(self, request: Request, organization) -> Response:
         project = result["project"]
         signal_first_monitor_created(project, request.user, False)
 
+        # alert_rule = request.data.get("alert_rule", {})
+        alert_rule = MonitorAlertRuleValidator(
```

Comment:
Yeah definitely just make this a sub-validator on the MonitorValidator and you won't have to do this here.